player/lua/ytdl_hook.lua

```diff
@@ -693,11 +693,21 @@ local function add_single_video(json)
     end
 
     -- set start time
-    if not (json.start_time == nil) and
+    if (not (json.start_time == nil) or not (json.section_start == nil)) and
```

Comment:
`not (var == nil)` can be replaced with `var`, because in lua anything that is not nil nor false is truthy.

`if (json.start_time or json.section_start) and ...`

Reply:
the rest of the file has quite a few instances of `if not (thing == nil)`. Should those be replaced as well?

Reply:
Alright then, I'll leave the rest as is.

Fixup commit squashed and ready to be merged.